game/templates/game/board.html

```
@@ -272,274 +330,188 @@
-            opacity: 0.7;
-        }
+<body>
 
-        .clock .time {
-            font-family: 'Consolas', 'Courier New', monospace;
-            font-size: 1.15rem;
-            font-weight: 700;
-        }
+    <!-- MOBILE FAB -->
+<div class="m-fab" id="mFab">
+  <button class="m-fab-btn" title="Pause"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)